src/tests/grpc/clients/ea_restaurant_java_etl_grpc_client_test.py

```diff
@@ -1,94 +1,171 @@
 import unittest
 from unittest import mock
 
+import grpc
+
+from src.env_config import get_env_config_instance
+from src.exceptions.exceptions import GrpcPermissionDeniedException
 from src.grpc.clients.ea_restaurant_java_etl_grpc_client import (
     EaRestaurantJavaEtlGrpcClient,
 )
 from src.grpc.clients.grpc_client import GrpcClient
 from src.tests.utils.fixtures.grpc_fixture import (
     build_login_client_response,
+    build_grpc_mongo_order_status_history,
+    build_uuids_response,
+    build_refresh_token_response,
     build_refresh_token_request,
 )
 from src.tests.utils.fixtures.mapping_odm_fixtures import build_order_status_history
 
+run_once = True
+
+
+def function_to_raise_exception_when_expired_access_token(
+    mongo_order_status_histories_request, metadata=None
+):
+    uuids_to_be_returned = build_uuids_response(
+        ["63656f20f2a8a6a247ae31cc", "63656f20f2a8a6a247ae31cd"]
+    )
+    global run_once
```

Comment:
Rename to `raise_exception_for_java_etl_client_insert_mongo_order_status_histories_from_python_etl_side_effect` instead.

Try to be very specific in this case if you are declaring global variables so you don''t have any conflicts

Reply:
Done.